backend/public-graph/graph/resolver.go

```diff
@@ -2309,62 +2309,94 @@ func (r *Resolver) AddTrackProperties(ctx context.Context, sessionID int, events
 				return e.New("error deserializing custom event properties")
 			}
 
-			if !strings.Contains(dataObject.Tag, "Track") {
+			trackEvent := strings.Contains(dataObject.Tag, "Track")
+			navigateEvent := strings.Contains(dataObject.Tag, "Navigate")
+			reloadEvent := strings.Contains(dataObject.Tag, "Reload")
+			clickEvent := strings.Contains(dataObject.Tag, "Click")
+
+			if !trackEvent && !navigateEvent && !reloadEvent && !clickEvent {
 				continue
 			}
 
 			if dataObject.Payload == nil {
-				return e.New("error reading raw payload from track event")
+				return e.New("error reading raw payload from session event")
 			}
 
-			var payloadStr string
-			if err := json.Unmarshal(dataObject.Payload, &payloadStr); err != nil {
-				return e.New("error deserializing track event payload into a string")
+			payloadStr := string(dataObject.Payload)
+			if !clickEvent {
+				if err := json.Unmarshal(dataObject.Payload, &payloadStr); err != nil {
+					return e.New("error deserializing session event payload into a string")
```

Comment:
Seem that click events were already in a form that would be unmarshalled to `map[string]string` rather than the initial `string` unmarshall